src/modules/src/power_distribution_stock.c

```diff
@@ -51,11 +52,11 @@ static struct {
   uint16_t m4;
 } motorPowerSet;
 
-#ifndef DEFAULT_IDLE_THRUST
-#define DEFAULT_IDLE_THRUST 0
+#ifndef MOTORS_DEFAULT_IDLE_THRUST
+#define MOTORS_DEFAULT_IDLE_THRUST 0
```

Comment:
This needs to be prepended with `CONFIG_`

Reply:
And the `#ifndef` dance is not needed, as noted below.

Reply:
Fixed in 0afe821

Reply:
It turns out that the `#ifndef` dance is needed because `CONFIG_MOTORS_DEFAULT_IDLE_THRUST` might be undefined if the drone is configured to start in armed mode (because in this case this option does not appear in `Kconfig`).